Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
A caveat seems fine but, then again, if it's not Homebrew specific I think users will probably just figure it out?